nncf/experimental/openvino_native/hardware/pattern_operations.py

```diff
@@ -41,5 +42,24 @@
                                                       ],
                          GraphPattern.LABEL_ATTR: 'ARITHMETIC'}
 
+ELEMENTWISE_OPERATIONS = {GraphPattern.METATYPE_ATTR: [ov_metatypes.OVAddMetatype,
```

Comment:
Could you please open the ticket to create one module to store all groups of metatypes?
There are several places where we define a group of metatypes
- https://github.com/openvinotoolkit/nncf/blob/37276693929f177aa44f39a8203a4b10bc8e175f/nncf/experimental/openvino_native/hardware/pattern_operations.py#L17-L24
- https://github.com/openvinotoolkit/nncf/blob/37276693929f177aa44f39a8203a4b10bc8e175f/nncf/experimental/openvino_native/graph/metatypes/common.py#L17-L23
- https://github.com/openvinotoolkit/nncf/blob/37276693929f177aa44f39a8203a4b10bc8e175f/nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py#L561-L566

It would be great to have only one module to store these groups. The purpose of the ticket is to provide a proposal how to fix it for OV backend.

Reply:
You can describe this problem in the 102414 ticket.